demisto_sdk/commands/common/hook_validations/lists.py

```diff
@@ -52,4 +51,8 @@ def is_valid_from_server_version(self) -> bool:
                                      file_path=None):
                     return False
             return True
-        return False
+        else:
```

Comment:
no need for the `else` statement after a return

please also fix the call to handle_error in line 50 (why `file_path` is None?)

Reply:
Didn't touch this code (handle_error) in that line, will change it